docs/administration.md

```diff
@@ -253,6 +253,65 @@ You can export your contents of your whole FAQ or just some selected categories
+
+The *id* column contains an ascending number that is assigned once and identifies each post. So for a new post you only need to count one further. This also applies to the *solution_id*, which can be used later to access individual FAQs. The keywords for the respective post should be entered in the *keywords* column, separated by commas. In the *topic* column you enter the question or the title of your post. The HTML-formatted content of the post should be entered in the *content* column. For the remaining columns, just use the existing entries as a guide. Check everything you have entered again. It's better to look at it once too often at the beginning until it becomes easier for you later. Save the table in exactly the same format as a CSV file, then you can close it.
+
+In order for the data from this first table to be displayed, the exact right values ​​in the other three tables are also required. Only then will it work and your data will also be displayed.
+
+You've probably already assigned some important *tags* to your existing posts, otherwise please do that now from within phpMyFAQ before we continue. It's best to just use one tag per post.
+
+So now let's take the second table *faqdata_tags.* On the left is the record_id, which matches the unique id from the previous table, but just is named differently. So for a new post you have to increment one here too. REMEMBER that this number refers to the exact post whose data you have already entered in the faqdata table! In the right-hand column there is a number that identifies the assigned tag.
+
+Within the Tags section of phpMyFAQ you can hover your mose over the red trash can and your browser will now show you a link at the bottom left that ends with our tag number. You should note down the tags and their numbers in a text file to have them at Hand later. With this knowledge we can now enter the desired tags in the table for our new posts. Check again and you can close the table.
+
+We continue with the third table, called *faqcategoryrelations*, and this is now about the categories. The number in the first column category_id identifies the category in which the new post should be sorted. You can assign the number to the category in exactly the same way as you did with the tags. So make a quick note in your text file! You can now enter the numbers so that the posts end up in the right category. On the right at record_id we have the unique ID that identifies the post. Once you have entered everything, save and close.
+
+The fourth file called *faqdata_user.csv* is easy. Just put in the *unique id* of your new entries at the left and *-1* at the right. Save and close.
```

Comment:
You need the faqdata_groups table as well, it's also just the ID of the entries and -1